qucs/AUTHORS

```diff
@@ -17,12 +17,29 @@
 -- Boston, MA 02110-1301, USA.  
 --
 
-Maintainer:
-	Michael Margraf <michael.margraf@alumni.tu-berlin.de>
+Original developers:
+	Michael Margraf
+	Stefan Jahn
 
-Authors: (see THANKS file for details)
-	Michael Margraf <michael.margraf@alumni.tu-berlin.de>
-	Stefan Jahn <stefan@lkcc.org>
-	Jens Flucke <jens.flucke@gmx.de>
-	Gopala Krishna A <krishna.ggk@gmail.com>
-	Gunther Kraut <gn.kraut@online.de>
+Current developers:
+	Guilherme Brondani Torri
+	Mike Brinson
+	Richard Crozier
+	Bastien Roucaries
+	Frans Schreuder
+	Vadim Kuznetsov
+	Claudio Girardi
+	Felix Salfelder
+	Andr�s Mart�nez Mera
```

Comment:
... Or even write (i, e) instead of (í, é). It is not so important.
On Jan 29, 2016 12:06 AM, "felix" <notifications@github.com> wrote:

> In qucs/AUTHORS
> <https://github.com/Qucs/qucs/pull/427#discussion_r51202930>:
>
> > -Authors: (see THANKS file for details)
> > -	Michael Margraf <michael.margraf@alumni.tu-berlin.de>
> > -	Stefan Jahn <stefan@lkcc.org>
> > -	Jens Flucke <jens.flucke@gmx.de>
> > -	Gopala Krishna A <krishna.ggk@gmail.com>
> > -	Gunther Kraut <gn.kraut@online.de>
> > +Current developers:
> > +	Guilherme Brondani Torri
> > +	Mike Brinson
> > +	Richard Crozier
> > +	Bastien Roucaries
> > +	Frans Schreuder
> > +	Vadim Kuznetsov
> > +	Claudio Girardi
> > +	Felix Salfelder
> > +	Andr�s Mart�nez Mera
>
> perhaps, we should just use utf8 here...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Qucs/qucs/pull/427/files#r51202930>.
>